devtools/replica-setup/checker.sh

```diff
@@ -37,16 +38,20 @@ check_server_params() {
         return 1
     fi
 
-    # Validate enforce_gtid_consistency
-    if [[ "$enforce_gtid_consistency" != "ON" ]]; then
-        echo "Error: enforce_gtid_consistency is not set to 'ON', it is set to '$enforce_gtid_consistency'."
-        return 1
-    fi
-
-    # Validate gtid_mode
-    if [[ "$gtid_mode" != "ON" ]]; then
-        echo "Error: gtid_mode is not set to 'ON', it is set to '$gtid_mode'."
-        return 1
+    # MariaDB use gtid_strict_mode instead of gtid_mode
+    # Validate gtid_strict_mode (for MariaDB)
+    if [[ "$gtid_strict_mode" != "ON" ]]; then
+        # Validate enforce_gtid_consistency (for MySQL)
+        if [[ "$enforce_gtid_consistency" != "ON" ]]; then
+            echo "Error: enforce_gtid_consistency is not set to 'ON', it is set to '$enforce_gtid_consistency'."
```

Comment:
This error message is not proper for MariaDB. It would be better to report a MariaDB-specific error message by detecting if `$gtid_strict_mode` is not empty but is not "ON".